lib/events.js

```diff
@@ -82,6 +83,13 @@ ObjectDefineProperty(EventEmitter, 'captureRejections', {
   enumerable: true
 });
 
+ObjectDefineProperty(EventEmitter, 'errorMonitor', {
+  value: kErrorMonitor,
+  writable: false,
+  configurable: true,
+  enumerable: true
```

Comment:
by adding an enumerable but nonwritable field, this is a potential breaking change - see https://github.com/googleapis/nodejs-spanner/pull/830#issuecomment-586477413.

In the future, we may want to consider enumerable fields as nonmajor only when they're normal data properties.

Reply:
Should we change this to a normal property? I used non writable simply because a write to this is usually a bug. I haven't seen any reasonable usecase to change this value but kept it configurable to allow any sort of hack if really needed.

In principle also a normal property added could be seen a breaking if some user added a property with the same name already before in it's code.

Reply:
Making it non-enumerable, or making it an accessor instead of a data property, would work as well.